docs/adrs/001-start-using-adrs.md

```diff
@@ -0,0 +1,42 @@
+---
+status: decided
+date: 17-05-2024
+deciders: Kamil Kołodziej
```

Comment:
**issue**: ADR format introduction should definitely be decided by a majority of the Fishjam team.